I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


The label _call_battle_presetup_2 is defined twice, at File "game/inanna n javier scenes.rpy", line 767:
                call battle_2_presetup from _call_battle_presetup_2
and File "game/kayden scenes.rpy", line 399:
            call battle_4_presetup from _call_battle_presetup_2


Ren'Py Version: Ren'Py 8.2.3.24061702
Mon Jul 29 19:19:06 2024
